src/main/cliapp/src/components/DataTableHeaderFooterTemplate.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import {Splitter, SplitterPanel} from "primereact/splitter";
+import { Button } from 'primereact/button';
+
+export function DataTableHeaderFooterTemplate({title, tableState, multiselectComponent, buttons, onclickEvent, isEnabled}){
+
+    return (
+        <div className="card">
```

Comment:
There is a primereact component that generates this:
<Card>
</Card>
I thing it would be better to use those when we can.

Reply:
This one --> import { Card } from 'primereact/card';  as in <Card> tag?

Reply:
Yes

Reply:
Sure, will change it.

Reply:
Just committed above change.

Reply:
I know there are other places in the code where we have `<div class="card">` but we can just change them out as we find them. @adamgibs